### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by jmrodri**
Adapted from the CNCF Code of Conduct.

### CODE REVIEW

**Comment by mhrivnak** [comment]
When you say this was "adapted", what exactly is different?

**Comment by mhrivnak** [comment]
Nevermind, I made a diff:

```diff
< ## CNCF Community Code of Conduct v1.0
---
> ## Automation Broker Community Code of Conduct v1.0
35c35
< Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a CNCF project maintainer, Sarah Novotny <sarahnovotny@google.com>, and/or Dan Kohn <dan@linuxfoundation.org>.
---
> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a Automation Broker project maintainer <ansible-service-broker@redhat.com>.
37,43c37,39
< This Code of Conduct is adapted from the Contributor Covenant
< (http://contributor-covenant.org), version 1.2.0, available at
< http://contributor-covenant.org/version/1/2/0/
< 
< ### CNCF Events Code of Conduct
< 
< CNCF events are governed by the Linux Foundation [Code of Conduct](http://events.linuxfoundation.org/events/cloudnativecon/attend/code-of-conduct) available on the event page. This is designed to be compatible with the above policy and also includes more details on responding to incidents.
---
> This Code of Conduct is adapted from the CNCF Code of Conduct v1.0 ( https://github.com/cncf/foundation/blob/master/code-of-conduct.md ),
> which is based on the Contributor Covenant ( http://contributor-covenant.org ), version 1.2.0, available at http://contributor-covenant.org/version/1/2/0/ ,
> and licensed under CC BY 4.0 ( https://creativecommons.org/licenses/by/4.0/legalcode ).
```

**Comment by fabianvf** [comment]
lgtm

**Comment by jmrodri** [comment]
@mhrivnak thanks for verifying the diff.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+comments, commits, code, wiki edits, issues, and other contributions that are not
+aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a Automation Broker project maintainer <ansible-service-broker@redhat.com>.
```
Comment (shawn-hurley) [inline CODE_OF_CONDUCT.md]: I would prefer to have specific people that someone could go to rather than a group-wide list.

**Comment by eriknelson** [comment]
So who do we want as the point of contact for this?

**Comment by jmrodri** [comment]
I'm working on getting an answer to whom should be the contact point.

**Pull request merged**